examples/cfq/train.py

```diff
@@ -338,6 +353,11 @@ def train_model(learning_rate: float = None,
     Do the training on data_source.train_dataset and evaluate on
     data_source.dev_dataset at each epoch and log the results
     """
+  if os.path.isdir(model_dir):
+    shutil.rmtree(model_dir)
```

Comment:
Could you add a comment what this is and why you do it?

Reply:
yes